packages/devtools/.vscode/launch.json

```diff
@@ -10,12 +10,14 @@
 			"preLaunchTask": "pub: build_runner serve"
```

Comment:
This project looks really nice! Can you update 
https://github.com/flutter/devtools/blob/master/CONTRIBUTING.md
to mention this as an easy way to develop the app?

Reply:
Yep, though I was going to move these files up to the root in another PR because opening the packages/devtools folder itself results in missing lints (analysis_options is in the root) so it's better to open the root folder. I'll add some notes on how to run from VS Code in that PR.